grafana-plugin/src/plugin.json

```diff
@@ -117,11 +117,7 @@
       "headers": [
         {
           "name": "X-Instance-Context",
-          "content": "{ \"grafana_token\": \"{{ .SecureJsonData.grafanaToken }}\", \"grafana_url\": \"{{ .JsonData.grafanaUrl }}\" }"
-        },
-        {
-          "name": "Authorization",
-          "content": "{{ .SecureJsonData.onCallInvitationToken }}"
+          "content": "{ \"grafana_token\": \"{{ .SecureJsonData.grafanaToken }}\" }"
```

Comment:
we no longer need to send the grafana URL to OnCall API, for the self-hosted install route, because this is now being configured directly in OnCall API